example/src/samples/sample-all-in-one.md

```diff
@@ -13,136 +6,66 @@
 ###### Heading 6
 
 
-## Emphasis
-
-### Example
-
 *Italic text*
-
 _Italic text_
 
 **Bold text**
-
 __Bold text__
 
 ***Bold and italic text***
-
 ___Bold and italic text___
 
 
-## Lists
-
-### Unordered List Example
-
 - Item 1
 - Item 2
   - Subitem 1
   - Subitem 2
 
 
-### Ordered List Example
-
-1. Item 1
-2. Item 2
-   1. Subitem 1
-   2. Subitem 2
-
+1. [x] Task 1
+2. [ ] Task 2
+3. [ ] Task 3
 
-## Links
-
-### Example
 
 [mynah-ui](https://github.com/aws/mynah-ui)
 
 <https://github.com/aws/mynah-ui>
 
 
-## Images
-
-### Example
-
 ![Alt text](https://placehold.co/600x400)
 
 
-## Blockquotes
-
-### Example
-
 > This is a blockquote.
 
 
-## Code
+`inline code`.
 
-### Inline Code Example
-Here is some `inline code`.
-
-### Block Code Example
-This is a block of code.
 ```
 no syntax declared
```

Comment:
Nice nit, can also be with 3 or more `~`. Should also do the check maybe with more than three `.

I'll add them on the next test implementation. However, it might also go to much testing a third part library.

Reply:
I remembered 4 spaces because we had bug with it.

Reply:
Exactly, especially for that we'll add them. But not like checking with 150 ` or ~ chars.